pandas/core/computation/expr.py

```diff
@@ -815,3 +822,5 @@ def names(self):
 
 
 _parsers = {"python": PythonExprVisitor, "pandas": PandasExprVisitor}
+
+
```

Comment:
this is going to cause the lint check in the CI to fail

Reply:
resolved this issue